### PR TITLE
Add workspace readme

### PR DESCRIPTION
**Pull request by michaeljguarino**
## Summary
Will be bootstrapped as the readme for installation repos

## Test Plan
n/a

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

### CODE REVIEW

File: cli-readme/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Welcome to Your Plural Repo
+
+This is the repo that will house all your plural installations for this cluster.  We'll give you a quick walkthrough how everything works here.  In general, there are two main steps w/ plural:
```
Comment (swoodward90) [inline cli-readme/README.md:3]: Capitalize Plural (and Console) throughout

File: cli-readme/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+In general we expect you to customize the repo as you wish, and any standard terraform or helm command should work as expected.  So if you, say, wanted to switch the database for an app to RDS, you can use the app's terraform folder to define the db and inject its secret into k8s right there.
+
+There are two other metadata files you should be aware of as well:
+
+* `context.yaml` - this is the initial app configuration you provide when running `plural bundle install`.  We use this to generate the resources that are present in your app folder above
+* `workspace.yaml` - high level workspace metadata information about your workspace, like the cloud account, cluster name, dns domain, etc
+
+## Secret Management
+
+We automatically encrypt this repo using a go reimplimentation of (git-crypt)[https://github.com/AGWA/git-crypt].  You'll be able to see the AES key you're currently using with `plural crypto export`.  We also drop the key fingerprint into the repo at `.keyid` to validate and provide appropriate error messages.
```
Comment (swoodward90) [inline cli-readme/README.md:55]: reimplementation

File: cli-readme/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  - main.tf # can override submodule configuration here, see that file for more info
+```
+
+In general we expect you to customize the repo as you wish, and any standard terraform or helm command should work as expected.  So if you, say, wanted to switch the database for an app to RDS, you can use the app's terraform folder to define the db and inject its secret into k8s right there.
+
+There are two other metadata files you should be aware of as well:
+
+* `context.yaml` - this is the initial app configuration you provide when running `plural bundle install`.  We use this to generate the resources that are present in your app folder above
+* `workspace.yaml` - high level workspace metadata information about your workspace, like the cloud account, cluster name, dns domain, etc
+
```
Comment (swoodward90) [inline cli-readme/README.md:52]: do you want to mention anything about editing context.yaml or workspace.yaml directly?

Comment (michaeljguarino) [reply cli-readme/README.md:53]: Yea but I'm not sure how to frame that. let me see

**Review by swoodward90 — APPROVED** [review]

**Pull request merged**